doc/modules/current_changes_header.template

```diff
@@ -6,6 +6,3 @@
  *
  *
  * <ol>
- *
- * </ol>
- */
```

Comment:
do we even still need this file, given that you've added `current_changes_header`?